client/javascripts/sagas/auth.js

```diff
@@ -0,0 +1,19 @@
+import { call, takeEvery } from 'redux-saga/effects';
```

Comment:
__[eslint]__
* Error - Resolve error
* Error - Missing file extension for "redux-saga/effects" (import/extensions)
* Error - Unable to resolve path to module 'redux-saga/effects'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->